components/snackbar.css

```diff
@@ -0,0 +1,59 @@
+:root {
+    --light-black-3: rgba(0, 0, 0, 0.2);
+    --grey-3: #333;
+    --white-color: #fff;
+    --button-text-color: #8b4edb;
+    --snack-opener-shade: rgba(115, 64, 182, 0.2);
+}
+
+#snack-opener {
+    background: var(--snack-opener-shade);
+    border-radius: 8px;
+}
+
+.snackbar {
+    position: fixed;
+    bottom: 10px;
+    left: 50%;
+    transform: translateX(-50%);
```

Comment:
If you are centering it on x-axis you can try margin: 0 auto;

Reply:
Oh that totally skipped my mind.